packages/contracts/contracts/registry/APMRegistry.sol

```diff
@@ -0,0 +1,35 @@
+/*
+ * SPDX-License-Identifier:    MIT
+ */
+
+pragma solidity 0.8.10;
+
+import "../core/component/ERC165Registry.sol";
+import "../core/IDAO.sol";
+import "../APM/IRepo.sol";
+
+/// @title Register plugin
+/// @author Sarkawt Noori - Aragon Association - 2022
+/// @notice This contract provides the possiblity to register a plugin repo by a unique address.
+contract APMRegistry is ERC165Registry {
```

Comment:
If `ERC165Registry` is the base class of APM Registry, I'm not sure if there's a better naming to illustrate this. Is APM an interface registry? I look at it as a place to register Plugin Repo's, and AFAIK all plugin repo's should have a common interface